packages/odyssey-storybook/test/testcafe/spec/button.e2e.ts

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import '../framework/pageobjects/buttonPageObject'
+import {buttonBuilder, getButtonSelector} from "../framework/pageobjects/buttonPageObject";
+
+const buttonVariants = [
+  { variant: 'primary'},
+  { variant: 'secondary'},
+  { variant: 'danger'},
+  { variant: 'floating'}
+];
+
+fixture('Testing disabled button');
+
+buttonVariants.forEach( c => {
+  test(
+    'when variant is set to '.concat(c.variant),
+    async browser => {
+      const button = buttonBuilder()
+        .setType('primary')
+        .setIsDisabled(true)
+        .setVariant(c.variant)
+        .build()
+
+      await browser
+        .navigateTo(
+          button.iFrameUrl()
+        )
+
+      await browser
+        .expect(
+          getButtonSelector().hasAttribute('disabled')
```

Comment:
I don't think we should use `getButtonSelector`.

If our components are rendering ARIA-safe with the correct accessibility flags, we should be doing `getByRole('button')` instead. Since this button's the only thing on the page, the test shouldn't break.